examples/run_tf_squad_with_trainer.py

```diff
@@ -0,0 +1,49 @@
+# coding=utf-8
+
+import os
+import logging
+
+logging.basicConfig(level=logging.INFO)
+
+import tensorflow as tf
+import tensorflow_datasets
+
+import torch
+
+from transformers import TFTrainer, BertForQuestionAnswering, BertTokenizer
+
+# Load dataset via TensorFlow Datasets
+data = tensorflow_datasets.load("squad")
+
+kwargs = {
+    "pretrained_model_name_or_path": "bert-base-cased",
+    "optimizer_name": "adamw",
+    "learning_rate": 3e-5,
+    "epsilon": 1e-08,
+    "loss_name": "squad_loss",
+    "batch_size": 8,
+    "eval_batch_size": 8,
+    "distributed": False,
+    "epochs": 2,
+    "data_processor_name": "squadv1",
+    "architecture": "TFBertForQuestionAnswering",
+    "max_len": 384,
+    "max_grad_norm": 1.0,
+    "doc_stride": 128,
+    "max_query_len": 64
+}
+
+trainer = TFTrainer(**kwargs)
```

Comment:
Yes, sorry my bad, all the pushed examples uses the old version of the trainer. I will update them and remove the Squad one because the QA task part is not yet implemented in the new version.